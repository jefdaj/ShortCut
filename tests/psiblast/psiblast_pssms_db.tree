.
|-- cache
|   |-- concat
|   |   |-- da426bef15.txt -> ../../cache/lines/5cbffd9463.txt
|   |   `-- emptybht.txt -> ../../cache/lines/47745ea386.txt
|   |-- each
|   |   |-- 7a152f8b02
|   |   |   |-- 074e75c6fb.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_074e75c6fb_86d15b07be.pssm
|   |   |   |-- 074e75c6fb.pssm.args -> ../../../cache/lines/26ff7bc6fa.txt
|   |   |   |-- 1cf587675f.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_1cf587675f_86d15b07be.pssm
|   |   |   |-- 1cf587675f.pssm.args -> ../../../cache/lines/c7bc4c1b7d.txt
|   |   |   |-- 31a1bb8e44.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_31a1bb8e44_86d15b07be.pssm
|   |   |   |-- 31a1bb8e44.pssm.args -> ../../../cache/lines/1aed11b39e.txt
|   |   |   |-- 452c75e69e.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_452c75e69e_86d15b07be.pssm
|   |   |   |-- 452c75e69e.pssm.args -> ../../../cache/lines/49b66e85b0.txt
|   |   |   |-- 6cff7e52c2.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_6cff7e52c2_86d15b07be.pssm
|   |   |   |-- 6cff7e52c2.pssm.args -> ../../../cache/lines/749a430fbd.txt
|   |   |   |-- 94e26f7884.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_94e26f7884_86d15b07be.pssm
|   |   |   |-- 94e26f7884.pssm.args -> ../../../cache/lines/2955bf81d5.txt
|   |   |   |-- 992e7fa31b.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_992e7fa31b_86d15b07be.pssm
|   |   |   |-- 992e7fa31b.pssm.args -> ../../../cache/lines/88016f7406.txt
|   |   |   |-- b0ecf6f058.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_b0ecf6f058_86d15b07be.pssm
|   |   |   |-- b0ecf6f058.pssm.args -> ../../../cache/lines/a5bb1a8bb8.txt
|   |   |   |-- c4ee1f6a9a.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_c4ee1f6a9a_86d15b07be.pssm
|   |   |   |-- c4ee1f6a9a.pssm.args -> ../../../cache/lines/0e72979a68.txt
|   |   |   |-- d8dbe517c2.pssm -> ../../../exprs/psiblast_train_pssms_db/9e754c0ec7_d8dbe517c2_86d15b07be.pssm
|   |   |   `-- d8dbe517c2.pssm.args -> ../../../cache/lines/82afdd4daf.txt
|   |   `-- d63861a51c
|   |       |-- 12fb00864f.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_12fb00864f_86d15b07be.bht
|   |       |-- 12fb00864f.bht.args -> ../../../cache/lines/07276ac025.txt
|   |       |-- 1b01341fc4.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_1b01341fc4_86d15b07be.bht
|   |       |-- 1b01341fc4.bht.args -> ../../../cache/lines/142b818711.txt
|   |       |-- 1ff655f857.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_1ff655f857_86d15b07be.bht
|   |       |-- 1ff655f857.bht.args -> ../../../cache/lines/c4a751b51f.txt
|   |       |-- 29308a8704.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_29308a8704_86d15b07be.bht
|   |       |-- 29308a8704.bht.args -> ../../../cache/lines/b2db2974e0.txt
|   |       |-- 5f5bdd614e.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_5f5bdd614e_86d15b07be.bht
|   |       |-- 5f5bdd614e.bht.args -> ../../../cache/lines/035363779e.txt
|   |       |-- 6db613714d.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_6db613714d_86d15b07be.bht
|   |       |-- 6db613714d.bht.args -> ../../../cache/lines/9bf6314472.txt
|   |       |-- 90f379dfc6.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_90f379dfc6_86d15b07be.bht
|   |       |-- 90f379dfc6.bht.args -> ../../../cache/lines/a3fc5a786b.txt
|   |       |-- d9c4fd8408.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_d9c4fd8408_86d15b07be.bht
|   |       |-- d9c4fd8408.bht.args -> ../../../cache/lines/3d4226b7af.txt
|   |       |-- eaae666ef5.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_eaae666ef5_86d15b07be.bht
|   |       |-- eaae666ef5.bht.args -> ../../../cache/lines/eca88d2294.txt
|   |       |-- efebd16e72.bht -> ../../../exprs/psiblast_pssm_db/9e754c0ec7_efebd16e72_86d15b07be.bht
|   |       `-- efebd16e72.bht.args -> ../../../cache/lines/506daf9a35.txt
|   |-- lines
|   |   |-- 035363779e.txt
|   |   |-- 07276ac025.txt
|   |   |-- 08add1fa44.txt
|   |   |-- 0cf27272d3.txt
|   |   |-- 0e72979a68.txt
|   |   |-- 142b818711.txt
|   |   |-- 1aed11b39e.txt
|   |   |-- 26ff7bc6fa.txt
|   |   |-- 2955bf81d5.txt
|   |   |-- 32ee44c616.txt
|   |   |-- 381fbbbdba.txt
|   |   |-- 3d4226b7af.txt
|   |   |-- 3f62a639a8.txt
|   |   |-- 47745ea386.txt
|   |   |-- 49b66e85b0.txt
|   |   |-- 4a02f07084.txt
|   |   |-- 506daf9a35.txt
|   |   |-- 5cbffd9463.txt
|   |   |-- 705d785b11.txt
|   |   |-- 749a430fbd.txt
|   |   |-- 7fed2c1eba.txt
|   |   |-- 82afdd4daf.txt
|   |   |-- 88016f7406.txt
|   |   |-- 8a8a1dc488.txt
|   |   |-- 932b4fb17d.txt
|   |   |-- 942fb825c9.txt
|   |   |-- 9bf6314472.txt
|   |   |-- 9ed52f0121.txt
|   |   |-- a3fc5a786b.txt
|   |   |-- a5bb1a8bb8.txt
|   |   |-- adfdb6bd8c.txt
|   |   |-- b0cae29378.txt
|   |   |-- b2db2974e0.txt
|   |   |-- bdc783547b.txt
|   |   |-- c3ae472116.txt
|   |   |-- c4a751b51f.txt
|   |   |-- c7bc4c1b7d.txt
|   |   |-- cd7881cdf9.txt
|   |   |-- eca88d2294.txt
|   |   |-- ef87d3e725.txt
|   |   `-- fda56c8059.txt
|   |-- load
|   |   |-- 32841598b2.faa -> $WORKDIR/data/7942genes10.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   `-- c3ae472116
|   |       |-- c3ae472116.pdb.phr
|   |       |-- c3ae472116.pdb.pin
|   |       `-- c3ae472116.pdb.psq
|   |-- psiblast
|   `-- split_faa
|-- exprs
|   |-- concat_bht
|   |   `-- d86ba85059.bht
|   |-- list
|   |   `-- 7d562183ce.faa.list -> ../../cache/lines/c3ae472116.txt
|   |-- load_faa
|   |   |-- 18
|   |   |   `-- a5
|   |   |       `-- 4f
|   |   |           `-- af
|   |   |               `-- ca
|   |   |                   `-- 4.faa
|   |   |-- 2d88028e40.faa -> ../../cache/load/32841598b2.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   |-- 55
|   |   |   `-- 89
|   |   |       `-- 5c
|   |   |           `-- bd
|   |   |               `-- 9e
|   |   |                   `-- e.faa
|   |   |-- 5b
|   |   |   `-- 31
|   |   |       `-- 60
|   |   |           `-- 91
|   |   |               `-- 63
|   |   |                   `-- 9.faa
|   |   |-- 6f
|   |   |   `-- 29
|   |   |       `-- 18
|   |   |           `-- 6e
|   |   |               `-- c9
|   |   |                   `-- 0.faa
|   |   |-- 82
|   |   |   `-- 64
|   |   |       `-- 5e
|   |   |           `-- d6
|   |   |               `-- f6
|   |   |                   `-- 1.faa
|   |   |-- 95
|   |   |   `-- 00
|   |   |       `-- e8
|   |   |           `-- 4b
|   |   |               `-- 09
|   |   |                   `-- 2.faa
|   |   |-- a3
|   |   |   `-- a4
|   |   |       `-- 22
|   |   |           `-- 67
|   |   |               `-- 17
|   |   |                   `-- 4.faa
|   |   |-- b2
|   |   |   `-- a2
|   |   |       `-- 61
|   |   |           `-- c0
|   |   |               `-- c1
|   |   |                   `-- f.faa
|   |   |-- ca
|   |   |   `-- b7
|   |   |       `-- 44
|   |   |           `-- a6
|   |   |               `-- f9
|   |   |                   `-- d.faa
|   |   |-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |   `-- e2
|   |       `-- 25
|   |           `-- e5
|   |               `-- 3d
|   |                   `-- 82
|   |                       `-- b.faa
|   |-- makeblastdb_prot_all
|   |   `-- a76d25f4b5.pdb -> ../../cache/lines/3f62a639a8.txt
|   |-- num
|   |   `-- 3e281fad0f.num -> ../../cache/lines/0cf27272d3.txt
|   |-- psiblast_each_pssm_db
|   |   `-- 5fbeb4e044_ef7788b175_e15f764d92.bht.list -> ../../cache/lines/b0cae29378.txt
|   |-- psiblast_pssm_db
|   |   |-- 9e754c0ec7_12fb00864f_86d15b07be.bht
|   |   |-- 9e754c0ec7_1b01341fc4_86d15b07be.bht
|   |   |-- 9e754c0ec7_1ff655f857_86d15b07be.bht
|   |   |-- 9e754c0ec7_29308a8704_86d15b07be.bht
|   |   |-- 9e754c0ec7_5f5bdd614e_86d15b07be.bht
|   |   |-- 9e754c0ec7_6db613714d_86d15b07be.bht
|   |   |-- 9e754c0ec7_90f379dfc6_86d15b07be.bht
|   |   |-- 9e754c0ec7_d9c4fd8408_86d15b07be.bht
|   |   |-- 9e754c0ec7_eaae666ef5_86d15b07be.bht
|   |   `-- 9e754c0ec7_efebd16e72_86d15b07be.bht
|   |-- psiblast_train_pssms_db
|   |   |-- 5fbeb4e044_a726be61de_e15f764d92.pssm.list -> ../../cache/lines/fda56c8059.txt
|   |   |-- 9e754c0ec7_074e75c6fb_86d15b07be.pssm -> ../../cache/lines/942fb825c9.txt
|   |   |-- 9e754c0ec7_1cf587675f_86d15b07be.pssm -> ../../cache/lines/32ee44c616.txt
|   |   |-- 9e754c0ec7_31a1bb8e44_86d15b07be.pssm -> ../../cache/lines/705d785b11.txt
|   |   |-- 9e754c0ec7_452c75e69e_86d15b07be.pssm -> ../../cache/lines/4a02f07084.txt
|   |   |-- 9e754c0ec7_6cff7e52c2_86d15b07be.pssm -> ../../cache/lines/7fed2c1eba.txt
|   |   |-- 9e754c0ec7_94e26f7884_86d15b07be.pssm -> ../../cache/lines/9ed52f0121.txt
|   |   |-- 9e754c0ec7_992e7fa31b_86d15b07be.pssm -> ../../cache/lines/8a8a1dc488.txt
|   |   |-- 9e754c0ec7_b0ecf6f058_86d15b07be.pssm -> ../../cache/lines/cd7881cdf9.txt
|   |   |-- 9e754c0ec7_c4ee1f6a9a_86d15b07be.pssm -> ../../cache/lines/08add1fa44.txt
|   |   `-- 9e754c0ec7_d8dbe517c2_86d15b07be.pssm -> ../../cache/lines/381fbbbdba.txt
|   |-- split_faa
|   |   `-- 1af4d085e7.faa.list -> ../../cache/lines/932b4fb17d.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- db90f32127.str -> ../../cache/lines/ef87d3e725.txt
|       `-- ec3b181d8a.str -> ../../cache/lines/adfdb6bd8c.txt
`-- vars
    |-- cyanodb.pdb -> ../exprs/makeblastdb_prot_all/a76d25f4b5.pdb
    |-- hittable.bht -> ../exprs/concat_bht/d86ba85059.bht
    |-- pcc6803.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- profiles.pssm.list -> ../exprs/psiblast_train_pssms_db/5fbeb4e044_a726be61de_e15f764d92.pssm.list
    |-- queries.faa.list -> ../exprs/split_faa/1af4d085e7.faa.list
    `-- result -> ../vars/hittable.bht

73 directories, 139 files
