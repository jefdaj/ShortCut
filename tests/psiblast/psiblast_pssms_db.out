Synpcc7942_0001	Synpcc7942_0001	100.000	390	0	0	193	582	1	390	0.0	785
Synpcc7942_0001	BAA16871.1	57.179	390	149	5	208	580	1	389	1.21e-148	433
Synpcc7942_0002	Synpcc7942_0002	100.000	290	0	0	193	482	1	290	0.0	586
Synpcc7942_0003	Synpcc7942_0003	100.000	777	0	0	193	969	1	777	0.0	1576
Synpcc7942_0003	BAA16646.1	74.809	786	178	9	187	966	4	775	0.0	1146
...

